### DISCUSSION TITLE
Can't install Homebrew: fatal: the remote end hung up unexpectedly31.81 MiB | 4.54 MiB/s fatal: early EOF fatal: index-pack failed Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master

### DISCUSSION THREAD

**Discussion by paupastur** [Homebrew/discussions#1019] [Getting started]
When I try to install Homebrew using `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

I get this error: 

`==> Tapping homebrew/core
remote: Enumerating objects: 77, done.
remote: Counting objects: 100% (77/77), done.
remote: Compressing objects: 100% (53/53), done.
fatal: the remote end hung up unexpectedly31.81 MiB | 4.54 MiB/s
fatal: early EOF
fatal: index-pack failed
Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master`

Please help! Also I am quite new to coding so please bear with me

**Comment by euoia** [comment]
See my comment in the other thread on this issue:
https://github.com/Homebrew/discussions/discussions/666#discussioncomment-489602
